### PR TITLE
Update examples.rst package name from Crypto to Cryptodome.

### PR DESCRIPTION
**Pull request by ymdatta**
When i tried to use according to the examples i got error(python version 3.6.3), i found out that its because of the package name.
It should be changed to Cryptodome instead of Crypto.
After changing the package name, everything works as expected.

### CODE REVIEW

**Comment by ymdatta** [comment]
Errors as Travis-cl confused which to choose between 3.6 and 3.6.3, checks should be upgraded to 3.6.3.

**Comment by Iceman9** [comment]
Hello. As stated in the installation guide

>1. a drop-in replacement for the old PyCrypto library

or

>2. a library independent of the old PyCrypto
 
The first one means that the name of the module is **Crypto** and in the second **Cryptodome**.

So the tutorial is valid, except there should be a ``if`` clause regarding how the user installs the module.

**Comment by ymdatta** [comment]
@Iceman9  Yeah, you are right.
But there is **no** PyCrypto package for **python3**, so i guess there should be something which says that if we want to use this in python3, it should be specifically be **Cryptodome** not **Crypto.**

What you said holds **good** for **python2** (pls correct me if i am wrong!).

**Comment by Varbin** [comment]
@ymdatta What do you mean there is no PyCrypto package for Py3? PyCrypto does support Python 3.

**Comment by Iceman9** [comment]
@ymdatta 
The guide refers to how you want the **namespace** for the *pycryptodome* chosen. AFAIK the PyCrypto package has not been updated since 2014. And now we have pycryptodome, which can provide more or less the same API (see the documentation about the differences), or if you are starting from scratch you can use the ``Cryptodome`` namespace.

If you install via command:
```
pip install pycryptodome
```
The namespace will be used such as:
```
from Crypto import ...
```
But if you use the command:
```
pip install pycryptodomex
```
The module will be imported as:
```
from Cryptodome import ...
```

So in short: Only the naming of the module is affected inside python.

Or I'm wrong and someone will correct me.
